src/utils/extra.ts

```diff
@@ -60,3 +63,45 @@ export const getAppExtra = <
 >(
   extra?: U,
 ): U[ItemType.APP] | undefined => extra?.[ItemType.APP];
+
+export const buildDocumentExtra = (
+  text: DocumentItemExtraProperties,
+): DocumentItemExtra => ({
+  [ItemType.DOCUMENT]: text,
+});
```

Comment:
Note: this will have to be updated after #83

Reply:
It is already taken care of by the fact that you changed the `DocumentItemExtraProperties` 😃

Reply:
Would it make sense to rename it though since the extra has more than just the text content?
```suggestion
export const buildDocumentExtra = (
  document: DocumentItemExtraProperties,
): DocumentItemExtra => ({
  [ItemType.DOCUMENT]: document,
});
```